src/images/Logo.svg

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" standalone="no"?>
```

Comment:
it seems that it would make sense to keep the logo SVG with the logo component itself.  Ideally, with components, everything it needs should ideally be co-located